docs/user.md

```diff
@@ -338,9 +333,21 @@ source types is given below.
             - cluster-node-2
   ```
 
+To add a new source type, a new handler specific to the source type is required. Please refer to the [developer manual][dev-doc] for more details.
 
+## Known issues
+- The NFS server ip and path are not validated, so please ensure that the servers are routable and paths are valid prior
+to the creation of the VolumeManager CR.
+In case an invalid `server` or `path` is used, Kubernetes publishes an event similar to the
+following during first attempt to use the PVC:
+```
+Unable to mount volumes for pod : timeout expired waiting for volumes to attach/mount for pod ...
+```
+
+- `timeoutForDataDownload` value should be of valid format as described [here](https://golang.org/pkg/time/#ParseDuration)
```

Comment:
This is not an issue. Also, it is already documented in https://github.com/kubeflow/experimental-kvc/blob/master/docs/user.md#types-of-sources.